mods/ra/maps/soviet-09/rules.yaml

```diff
@@ -0,0 +1,121 @@
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, soviet09.lua, soviet09-AI.lua
+	MissionData:
+		BriefingVideo: soviet9.vqa
+		WinVideo: v2rocket.vqa
+		LossVideo: sfrozen.vqa
+		StartVideo: movingin.vqa
+		Briefing: The Allied forces have intercepted and destroyed a convoy that carried parts for our secret weapon. One truck remains, but they have captured that last truck and its cargo.\n\nThis is not acceptable! You are to destroy that truck before the Allies leave the area with it.
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+	SmudgeLayer@SCORCH:
+		InitialSmudges:
+			98,87: sc3,0
+			100,87: sc2,0
+			108,87: sc1,0
+			88,93: sc6,0
+			104,101: sc3,0
+	SmudgeLayer@CRATER:
+		InitialSmudges:
+			99,87: cr1,2
+			86,93: cr1,0
+			87,93: cr1,1
+
+Player:
+	PlayerResources:
+		DefaultCash: 8000
+
+AFLD:
+	AirstrikePower@parabombs:
+		Prerequisites: ~disabled
+
+ATEK:
+	Buildable:
+		Prerequisites: ~disabled
+
+HPAD:
```

Comment:
Stole an Allied conyard and wasn't able to build a helipad, though I did capture one next to it. Was it disabled intentionally? (tech level?)

Reply:
I had intended to address this in my heli pr with an allies hpad and soviet hpad. I guess I could make the prereq allied tech.